split Eltex-mes OS to mes21xx / mes23xx. + few improvements

Npeca75 opened the pull request:
looks like ELTEX mibs/oids for 21xx and 23xx series are different so 21xx series could not be added in existing eltex-mes OS

Split OS to two branch
1. Eltex-mes21xx
2. Eltex-mes23xx

few improvements
1. Use **pre_cache** for discovery. This way, discovery time vas reduced from 79 sec to 28 sec and stress on CPU is smaller because **rlPhyTestGetResult** was called multiple times for each SFP sensor in first release
2. Use properly named sensors, following the names in MIBs
3. removed hardcoded OIDs from discovery files, using MIBs this time

tested on:
mes2324b
mes2308r
mes2124m


DO NOT DELETE THE UNDERLYING TEXT

#### Please note

> Please read this information carefully. You can run `./lnms dev:check` to check your code before submitting.

- [x] Have you followed our [code guidelines?](https://docs.librenms.org/Developing/Code-Guidelines/)
- [ ] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.
- [x] If my Pull Request makes discovery/polling/yaml changes, I have added/updated [test data](https://docs.librenms.org/Developing/os/Test-Units/).

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

Jellyfrog commented:
Why are vendors like this? :(
Great work tho,
Could you remove all MIBS not used?

Npeca75 commented:
hi @Jellyfrog 
> Why are vendors like this?
if i get it correctly, they using Marvell, Broadcom, and lately Realtek chips
so i expect to soon or later some ne eltex-mex xxxx variation will show up

>Great work tho,
thank you,still learning a lot about LNMS internals, GIT, and nice PHP coding

>Could you remove all MIBS not used?
i think i added them by purpose, one is connecting to another, but no problem
i will start to delete one by one in two new branch and see which one is not used

from "old" eltex-mes (mib/eltex) i removed few which i am put there few month ago in first PR, but will check again. Not so brave to delete something which belong to eltex-olt

Npeca75 commented:
@Jellyfrog 
deleted old MIBs
ready for review

Jellyfrog commented on includes/discovery/sensors/current/eltex-mes21xx.inc.php:
```diff
@@ -1,6 +1,6 @@
 <?php
```
It looks like these can be done in yaml, but that we can leave for another day™

Npeca75 replied:
sorry @Jellyfrog  but no, it could not be done in YAML
OIDs are not ending with Index
example:
as you could see, index (57, 58) is in the middle

rlPhyTestGetResult.57.rlPhyTestTableCablePairSkew1 = 8
rlPhyTestGetResult.57.rlPhyTestTableCablePairSkew2 = 0
rlPhyTestGetResult.57.rlPhyTestTableCablePairSkew3 = 0
rlPhyTestGetResult.57.rlPhyTestTableCablePairSkew4 = 0
rlPhyTestGetResult.58.rlPhyTestTableTransceiverTemp = 43
rlPhyTestGetResult.58.rlPhyTestTableTransceiverSupply = 3338400
rlPhyTestGetResult.58.rlPhyTestTableTxBias = 11150
rlPhyTestGetResult.58.rlPhyTestTableTxOutput = -6068
rlPhyTestGetResult.58.rlPhyTestTableRxOpticalPower = -7129
rlPhyTestGetResult.58.rlPhyTestTableDataReady = 1
rlPhyTestGetResult.58.rlPhyTestTableLOS = 0
rlPhyTestGetResult.58.rlPhyTestTableTxFault = 0

Jellyfrog replied:
Why follow a standard when you can ignore it

Jellyfrog commented:
Good work here!
